drivers/debian/etc/modprobe.d/tier4-imx490.conf

```diff
@@ -9,7 +9,15 @@
 # "enable_distortion_correction=0" disable distortion correction feature. (Default)
 # "enable_distortion_correction=1" enable distortion correction feature.
 #
+# "fsync_mode=0" disable FPGA Fsync mode. (Default)
```

Comment:
Could you add a note that it is only valid for RQX-58G/RQX-59G?

Reply:
Sure, please check this commit: https://github.com/tier4/tier4_automotive_hdr_camera/pull/12/commits/41dccf226e9557654336c70ab39dd0311a75dc68
Thank you!